src/index.js

```diff
@@ -7,8 +7,8 @@ if (!('import' in document.createElement('link'))) {
   require('../lib/vendor/HTMLImports');
 }
 
-require('../style/aframe-core.css');
-require('../style/rStats.css');
+require('./style/aframe-core.css');
+require('./style/rStats.css');
```

Comment:
you need to fix this path for rStats:

https://github.com/aframevr/aframe/blob/dev/src/core/a-scene.js#L621